src/app/src/main/kotlin/org/imperial/mrc/hint/logging/GenericLoggerImpl.kt

```diff
@@ -0,0 +1,37 @@
+package org.imperial.mrc.hint.logging
+
+import net.logstash.logback.argument.StructuredArguments.kv
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Component
+
+@Component
+class GenericLoggerImpl(private val logger: Logger = LoggerFactory.getLogger(GenericLoggerImpl::class.java)): GenericLogger
+{
+
+    override fun info(log: LogMetadata)
+    {
+        logger.info("TEST INFO",
```

Comment:
This "TEST INFO" is meant to be a format string, according to Logger docs - what's the plan for this (and for the error case too)?

Reply:
This is more like a placeholder for this MVP but I have added a more reasonable message.